tools/marvin/marvin/config/Baremtal_AdvZone_Vmware.cfg

```diff
@@ -0,0 +1,191 @@
+                    "vlan": "1220",
+                    "gateway": "10.x.x.1"
+                }
+            ],
+            "networktype": "Advanced",
+            "pods": [
+                {
+                    "endip": "10.x.x.x",
+                    "name": "POD01",
+                    "startip": "10.x.x.x",
+                    "netmask": "255.255.255.192",
+		    "vmwaredc":{"name":"dc-bare","vcenter":"10.x.x.x","username":"Administrator@vsphere.local","password":"Password"},
+                    "clusters": [
+                        {
+                            "clustername": "10.x.x.x/dc-bar/cluster1",
+                            "hypervisor": "VmWare",
+                            "username": "Administrator@vsphere.local",
+                            "clustertype": "ExternalManaged",
+                            "password": "Password",
+                            "url": "http://10.x.x.x/dc-bare/cluster1",
+                            "hosts": [
+                                {
+                                    "username": "root",
+                                    "url": "http://10.x.x.x",
+                                    "password": "password"
+                                },
+                                {
+                                    "username": "root",
+                                    "url": "http://10.x.x.x",
+                                    "password": "password"
+                                }
+                            ],
+                            "primaryStorages": [
+                                {
+                                    "url": "nfs://10.x.x.x:/export/home/sadhu/primrc2",
+                                    "name": "PS0"
+                                }
+                            ]
+                        },
+						{
+                            "clustername": "C1",
+                            "hypervisor": "BareMetal",
+                            "hosts": [
+                                {
+                                    "username": "root",
+                                    "url": "http://10.x.x.x",
+                                    "password": "password",
+                                    "hostmac": "b2:ac:21:s2:1c:20",
+                                    "cpunumber": "1",
+                                    "cpuspeed": "1000",
+                                    "memory": "1024",
+									"hosttags": "host15"
+                                }
+                            ],
+                            "clustertype": "CloudManaged"
+                        }
+                    ],
+                    "gateway": "10.x.x.x"
+                }
+            ],
+            "internaldns1": "10.x.x.x",
+			"baremetalrcturl": "http://10.x.x.x/baremetal/sadhurct1.json_old",
+            "secondaryStorages": [
+                {
+                    "url": "nfs://10.x.x.x:/export/home/sadhu/secrc2",
```

Comment:
its good if you can remove your name in path both in secondary and primary storage path